docs/en/latest/plugins/prometheus.md

```diff
@@ -27,438 +27,423 @@ description: This document contains information about the Apache APISIX promethe
+| instance_id | Available only in Enterprise. ID of the gateway instance that latencies correspond to. |
+| api_product_id | Available only in Enterprise. Product ID that latencies correspond to. |
+
+#### Latency Types
+
+`apisix_http_latency` can be labeled with one of the three types:
+
+* `request` represents the time elapsed between the first byte was read from the client and the log write after the last byte was sent to the client.
+
+* `upstream` represents the time elapsed waiting on responses from the upstream service.
+
+* `apisix` represents the difference between the `request` latency and `upstream` latency.
+
+In other words, the APISIX latency is not only attributed to the Lua processing. It should be understood as follows:
+
+```text
+APISIX latency
+  = downstream request time - upstream response time
+  = downstream traffic latency + NGINX latency
 ```
 
-### Specifying `metrics`
-
-For http request related metrics, you could specify extra labels, which match the APISIX variables.
+### Labels for `apisix_upstream_status`
 
-If you specify label for nonexist APISIX variable, the label value would be "".
+The following labels are used to differentiate `apisix_upstream_status` metrics.
 
-Currently, only below metrics are supported:
+| Name | Description                                                                                         |
+| ---------- | --------------------------------------------------------------------------------------------------- |
+| name       | Resource ID corresponding to the upstream configured with health checks, such as `/apisix/routes/1` and `/apisix/upstreams/1`. |
+| ip         | IP address of the upstream node.                                                                         |
+| port       | Port number of the node.                                                                            |
 
-* http_status
-* http_latency
-* bandwidth
+## Examples
 
-Here is a configuration example:
-
-```yaml title="conf/config.yaml"
-plugin_attr:
-  prometheus:
-    metrics:
-        http_status:
-            extra_labels:
-                - upstream_addr: $upstream_addr
-                - upstream_status: $upstream_status
-```
+The examples below demonstrate how you can work with the `prometheus` plugin for different scenarios.
 
-### Specifying `default_buckets`
+### Get APISIX Metrics
 
-`DEFAULT_BUCKETS` is the default value for bucket array in `http_latency` metrics.
+The following example demonstrates how you can get metrics from APISIX.
 
-You can change the `DEFAULT_BUCKETS` by configuring `default_buckets` attribute in you configuration file.
+The default Prometheus metrics endpoint and other Prometheus related configurations can be found in the [static configuration](#static-configurations). If you would like to customize these configuration, see [configuration files](/apisix/reference/configuration-files#configyaml-and-configyamlexample).
 
```

Comment:
no related content

Reply:
It's in `config.yaml`. You can mention it in a separate section - this is what other plugin docs here do.